packages/core/src/elements/FormFieldElement.ts

```diff
@@ -386,4 +404,44 @@ export abstract class FormFieldElement extends ControlElement {
   ): void {
     this.inputElement?.setSelectionRange(startSelection, endSelection, selectionDirection);
   }
+
+  /**
+   * Template for clear button
+   * Rendered when `clears` attribute is set
+   * @returns Popup template or undefined
+   */
+  protected get clearButtonTemplate(): TemplateResult | typeof nothing {
+    if (this.hasClear) {
+      return html`
+        <div
+          ${ref(this.clearsButtonRef)}
+          id="clears-button"
+          part="button button-clear"
+          @tap="${this.onClearsButtonTap}"
+        >
+          <ef-icon part="icon icon-clear" icon="cross"></ef-icon>
+        </div>
+      `;
+    }
+    return nothing;
+  }
+
+  /**
+   * Run when tap event happens on clears button and fire value-changed event.
+   * @param event tap event
+   * @returns {void}
+   */
+  protected onClearsButtonTap(event: TapEvent): void {
+    this.setValueAndNotify('');
+    event?.preventDefault();
```

Comment:
No need for optional chaining.
```suggestion
    event.preventDefault();
```

Reply:
let's also add a comment explaining why `preventDefault()` is needed.